src/lib/cfa/ntp_conf.rb

```diff
@@ -212,27 +211,38 @@ def value=(value)
 
       def comment
         return nil unless tree_value?
-        tree_value.tree["#comment"]
+        tree_value.tree["#comment[]"]
       end
 
       def comment=(comment)
-        create_tree_value unless tree_value?
+        ensure_tree_value
         if comment.to_s == ""
-          tree_value.tree.delete("#comment")
+          tree_value.tree.delete("#comment[]")
         else
-          tree_value.tree["#comment"] = comment
+          tree_value.tree["#comment[]"] = comment
```

Comment:
if there is more comments which one will be overwritten? I think this part need test coverage of this functionality. So we are sure how it exactly works and can document it.